conduit/data/datasets/vision/nico.py

```diff
@@ -131,8 +130,8 @@ def train_test_split(
         *,
         train_props: Optional[Dict[Union[str, int], Dict[Union[str, int], float]]] = None,
         seed: Optional[int] = None,
-    ) -> NICOTrainTestSplit:
-        """Split the data into train/test sets with the option to condition on concept/context."""
+    ) -> TrainTestSplit["NICO"]:
```

Comment:
```suggestion
    ) -> TrainTestSplit[NICO]:
```

Reply:
This doesn't work since we're not using ``annotations``.